app/uk/gov/hmrc/entrydeclarationdecision/services/ProcessDecisionService.scala

```diff
@@ -90,7 +89,9 @@ class ProcessDecisionService @Inject()(
         }
 
       processDecisionResponse(decision.response).andThen {
-        case Success(Right(_)) => storeConnector.setShortTtl(decision.submissionId)
+        case Success(Right(_)) =>
+          isLongJourneyTime(decision.metadata.receivedDateTime, appConfig.longJourneyTime)
```

Comment:
This does read like a side effect. 

Why not move the logic into this service class? And perhaps use the existing `PagerDutyLogger` so that this side-effect requirement can be tested.